### ISSUE TITLE
Currently clicking on the thumbnail acts as an expando for images, but for links, it opens the link

### ISSUE THREAD

**Issue by Kommynct** [LemmyNet/lemmy-ui#1142]
This is problematic because as i'm scrolling through, it's sometimes annoying to have the page i'm on replaced when i'm expecting an expando to expand, I think there's two possible solutions:

1. Move expandos somewhere else
2. make left clicking thumbnails open in a new tab

Thoughts?

**Comment by krestenlaust** [comment]
I vote option 2.

- The pictogram used for external links indicate opening in a new tab (imo)
- I can't remember when I last encountered a popular site open external links in the same tab (somewhat breaks the expected rules of the internet)

**Comment by krestenlaust** [comment]
Issue already exists #634

**Comment by mwalser** [comment]
I agree that the current behavior is really confusing. While there are icons to discern an image from a link post, its easy to miss.

My suggestion would be to make two changes:
- Clicking an image always navigates to the image or article URL and never opens an expando
- For image posts, introduce a button to open an expando

That way its consistent:
- Clicking on the image always opens the source URL
- Clicking on the title always opens the comments
- Clicking on the "+" icon always opens an expando

**Comment by Kommynct** [comment]
That wouldn't encourage me to participate, that would just be annoying, but if that argument can be made, it should be up to the user, or the instance.

**Comment by krestenlaust** [comment]
> That wouldn't encourage me to participate, that would just be annoying, but if that argument can be made, it should be up to the user, or the instance.

Then I'd rather back up a bit, and just forget about that. I don't believe that making a new UI setting every time we don't completely agree, is the right call.

**Comment by doctorrmcb** [comment]
I would use the site more if it had expandos for text and other posts. Any time you take me to another page it is really annoying. I want the primary viewing experience to be how it is in RES, where I click through expandos to get content quickly and if I'm interested in a discussion then I will open the comments. There is way too much loading when I have to open other pages to view content and going back is painful.

**Comment by krestenlaust** [comment]
Agree. The way you have to navigate backwards is a real nuisance currently. I try to use the context menu or Ctrl+Left arrow, but that's just a short-term solution.

Isn't it possible to expand text as it is now anyway?

**Comment by doctorrmcb** [comment]
If it is possible it isn't apparent to me how to do it. I found the image functionality easy enough to find, but after 30 minutes of using Lemmy, I got frustrated and gave up using it. Primarily due to another bug, but the inconsistency of where to click to expand things would definitely be another reason.

**Comment by krestenlaust** [comment]
I have high hopes for the next release, it should fix a lot of the annoying bugs, and some of the annoying UX
